README.md

```diff
@@ -687,7 +687,7 @@ at `DEBUG` level.
 
 ## Deleting pods in bad state
 
-    kubectl get pods -o name --selector=jenkins=slave --all-namespaces  | xargs -I {} kubectl delete {}
+    kubectl get pods -o name --selector=jenkins=agent --all-namespaces  | xargs -I {} kubectl delete {}
```

Comment:
needs a code update.

Reply:
I will skip this change in this iteration and do more work down the line to address this